### ISSUE TITLE
Collective Navigation v2: create new Collective page navbar

### ISSUE THREAD

**Issue by sbinlondon** [opencollective/opencollective#3639]
Component in desktop, tablet, mobile:

![Screenshot 2020-10-28 at 12 15 08](https://user-images.githubusercontent.com/37520401/97434768-66870900-1917-11eb-9be1-3511a63627f4.png)

![Screenshot 2020-10-28 at 12 15 15](https://user-images.githubusercontent.com/37520401/97434751-61c25500-1917-11eb-91a9-bb83edc9aaaf.png)

![Screenshot 2020-10-28 at 12 15 21](https://user-images.githubusercontent.com/37520401/97434755-625aeb80-1917-11eb-955d-1f172ce37834.png)

![image](https://user-images.githubusercontent.com/37520401/101030856-5810ba80-3571-11eb-978c-ef92a8f59f38.png)

Component as seen on the Collective page:

![Screenshot 2020-10-28 at 12 15 58](https://user-images.githubusercontent.com/37520401/97434763-64bd4580-1917-11eb-8650-e4265da321da.png)

![Screenshot 2020-10-28 at 12 15 49](https://user-images.githubusercontent.com/37520401/97434760-638c1880-1917-11eb-8d5d-2498b5b0c93b.png)

Notes:

* subcomponents: 
    * "actions" menu https://github.com/opencollective/opencollective/issues/3640
    * multiple section drop-down menus https://github.com/opencollective/opencollective/issues/3641
* important to note difference between desktop/tablet and mobile menu (i.e. "actions" is a popper menu on desktop/tablet, but an expandable section on mobile)

**Comment by Betree** [comment]
Given that we want the navbar to look the same on multiple pages, it means we need a way to fetch the same data every time, on all pages. I really think that we should achieve that by implementing a dedicated query on API, as described in https://github.com/opencollective/opencollective/issues/2807. 

I've never merged it because I was hopping for more feedback before doing so, but the feature was actually implemented in https://github.com/opencollective/opencollective-api/pull/3992. I can rebase the PR & merge if you're ready to use that.

**Comment by sbinlondon** [comment]
> Given that we want the navbar to look the same on multiple pages, it means we need a way to fetch the same data every time, on all pages. I really think that we should achieve that by implementing a dedicated query on API, as described in #2807.
> 
> I've never merged it because I was hopping for more feedback before doing so, but the feature was actually implemented in [opencollective/opencollective-api#3992](https://github.com/opencollective/opencollective-api/pull/3992). I can rebase the PR & merge if you're ready to use that.

Hi Ben just looking at this again, I think this would be a great improveent - can you please rebase and merge the PR as suggested and let me know how it's meant to be used once you're back?

**Comment by sbinlondon** [comment]
Some clarifications in discussion with design:

* The back arrow and Collective avatar icon: this is displayed in the navbar on subpages (like `/collectiveslug/budget/transactions`) so it will take you back to the main Collective page.

**Comment by Betree** [comment]
@sbinlondon I'll do that :+1: The behavior is described in https://github.com/opencollective/opencollective/issues/2807. Basically, you just query the `features` field like that:

```gql
Collective {
  features {
    UPDATES
  }
}
```

And get something like:
```json
{
  "collective": {
    "features": {
      "UPDATES": "DISABLED",
    }
  }
}
```

**Comment by sbinlondon** [comment]
Features endpoint has now been merged. Starting to use it on the frontend in https://github.com/opencollective/opencollective-frontend/pull/5415

To do next: update the graphql prefetching and sections data fetching for the collective page to only fetch data for features like UPDATES or CONVERSATIONS if they are supported and active

**Comment by Betree** [comment]
> To do next: update the graphql prefetching and sections data fetching for the collective page to only fetch data for features like UPDATES or CONVERSATIONS if they are supported and active

Could we specify that in an issue and solve it later? It's a potential good performance improvement, but:
- Performances improvements have to be measured. Maybe it's actually worse to do it this way because we'll be triggering multiple GraphQL queries in parallel (thus loaders won't be able to do their job)
- It doesn't seem like a priority in the context of this project

Unless there's another argument than perf that I'm not seeing here.

**Comment by sbinlondon** [comment]
> > To do next: update the graphql prefetching and sections data fetching for the collective page to only fetch data for features like UPDATES or CONVERSATIONS if they are supported and active
> 
> Could we specify that in an issue and solve it later?

It is a [separate issue](https://github.com/opencollective/opencollective/issues/3699)! 👍

**Comment by sbinlondon** [comment]
Now the overall navbar v2 structure has been created and it's just specific things to implement/polish, I'm closing this issue and opening more targeted ones so we can keep track better.
